### ISSUE TITLE
Label keeps getting deleted from a component

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#322]
The world is https://matt.engagelively.com/users/rick/published/DANResponsive/index.html.  The left-hand card is supposed to have a label, which I've added multiple times, even deleting and re-attaching the component.  The label keeps disappearing -- clearly something in the component architecture

**Comment by merryman** [comment]
Looking at your world, I can drop labels anywhere I want. I can't really reproduce the behavior you have decribed.

**Comment by rickmcgeer** [comment]
I can add the labels just fine, and then when I save the world and reload they disappear.
To reproduce:
Add a label "Months Since Last Dive" to the experienceBox on the right-hand third box in the main screen:
![image](https://user-images.githubusercontent.com/656259/132516800-4839a011-b1ec-425d-8ad8-c4f50b1e13b7.png)
Added....
![image](https://user-images.githubusercontent.com/656259/132517571-9a5e9f9e-c63b-494d-a53a-664883e5cab0.png)
Save the world and reload, and the label has disappeared:
![image](https://user-images.githubusercontent.com/656259/132518435-77cd9b36-ab72-4029-bd2e-38db7c701cbd.png)
The experienceBox is an instance of a component, which does have the label.  But even when I try deleting it and then resaving with the label, the same thing happens
![image](https://user-images.githubusercontent.com/656259/132519053-38e0dbba-f556-4744-bbee-4088a3fceca8.png)

**Comment by rickmcgeer** [comment]
Now works.  I had instantiated an intermediate component, wide container, which contained an old version (a copy, not the component itself) of experienceBox, so changes I made were getting scrubbed by the intermediate container.  
This leads to a question about best practices for component architecture and component use.  I screwed up here, but the really interesting question is how can we document/make tools available for people so they won't make the same mistake I did. 
Assigning to @merryman for any comment and close.
